### PR TITLE
Exposes port 9090:8080 in the container

### PR DESCRIPTION
**Pull request by sywhb**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -140,8 +140,8 @@ services:
       context: .
       dockerfile: ./packages/content-fetch/Dockerfile-local
```
Comment (sywhb) [inline docker-compose.yml:141]: @jacksonh why are we using a `Dockerfile-local` here? Is it something different from the `Dockerfile`?

Comment (jacksonh) [reply docker-compose.yml:141]: @sywhb i think there was some issue running Dockerfile locally with docker-compose vs Dockerfile which was running in App Engine. 

Probably an issue with the Chromium dependency

Comment (sywhb) [reply docker-compose.yml:141]: I didn't have such an issue. Maybe it is good to remove the `Dockerfile-local` now?

Comment (jacksonh) [reply docker-compose.yml:141]: @sywhb yeah probably ok now. Maybe need to do a full test with --force-recreate also

**Review by jacksonh — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (sywhb) [reply docker-compose.yml:141]: Yeah, I tested

**Pull request merged**